securedrop_client/gui/widgets.py

```diff
@@ -253,6 +259,36 @@ def toggle_star(self, event):
         """
         self.controller.update_star(self.source)
 
+    def delete_source(self, event):
+        """It will launch the message box.
+
+        The Message box will warns the user regarding the seniority of the
+        operation. It will re-confirm the desire to delete the source. On
+        positive answer, it will delete the record of source both from
+        SecureDrop server and local state.
+        """
+        message = (
+            "<big>Deleting the Source account for",
+            "<b>%s</b> will also" % (self.source.journalist_designation,),
+            "delete %d files and %d messages.</big>" % (
+                len(self.source.submissions), len(self.source.replies)
+            ),
+            "",
+            "<small>This Source will no longer be able to correspond",
+            "through the log-in tied to this account.</small>",
+        )
+        message = ' <br> '.join(message)
```

Comment:
I don't know how Qt handles this, but it might be better to not have manual `<br>`'s but instead get the text to wrap. If possible?

Reply:
Removed custom `<br>` tags. I was using them for exactly replicating the [wireframe](https://projects.invisionapp.com/share/EUOVALO94RZ).

For separating both (large and small) messages, I am unable to find a solution without using a `<br>` tag. If you have any batter solution, then please suggest. If not, then I request you to resolve the conversation. Thanks!

Reply:
I think it's ok to use it to separate the two paragraphs, but I didn't want to use it for formatting each line.